cli/tests/testdata/run/deny_some_permission_args.out

```diff
@@ -0,0 +1,22 @@
+PermissionStatus { state: "granted-partial", onchange: null }
```

Comment:
I think this should be represented as `{ state: "granted", partial: true }` to not break compatibility with `{ state: "granted" }` and because for some use cases they are both acceptable results.

Reply:
Ah! Yes. Fix incoming.